Add note on handling line endings for Windows users (#781)
eisop/checker-framework: docs/manual/troubleshooting.tex

You might want to add an \<export CHECKERFRAMEWORK=...> line to your
\<.bashrc> file.

> \textbf{Ensure Line Endings are Handled Properly}: Run
> \texttt{git config --global core.autocrlf true} in Git Bash to prevent line ending
> issues between Windows and Unix-based systems, or execute Git commands directly
> from the Ubuntu terminal.

Reviewer: ```suggestion
\textbf{Ensure line endings are handled properly}: Run
\code{git config --global core.autocrlf true} to prevent line ending issues between
Windows and Unix-based systems.
```

Reviewer: The `Ubuntu terminal` part is too distribution specific. Maybe there should be a general note about how to build on Windows/Unix?

You might want to add an \<export CHECKERFRAMEWORK=...> line to your
\<.bashrc> file.

> \textbf{Ensure Line Endings are Handled Properly}: Run

Reviewer: ```suggestion
Ensure line endings are handled properly. If necessary, run
```
